python/lsst/daf/base/baseLib.i

```diff
@@ -161,6 +161,48 @@ def _propertyContainerGet(container, name, asArray=False):
         pass
     raise lsst.pex.exceptions.TypeError('Unknown PropertySet value type for ' + name)
 
+import numbers
+import sys
+def _guessIntegerType(container, name, value):
+    """Given an existing container and name, determine the type
+    that should be used for the supplied value. The supplied value
+    is assumed to be a scalar.
+
+    On Python 3 all ints are LongLong but we need to be able to store them
+    in Int containers if that is what is being used (testing for truncation).
+    Python3 uses sys.maxsize as an equivalence for the cross over from int to long.
+
+    If there is no pre-existing value we have to decide what to do. For now
+    we pick Int if the value is less than maxsize.
+
+    On Legacy Python always returns None.
```

Comment:
Oh, I see.  "On Legacy Python, always returns None."  (comma).  Or "Always returns None for Legacy Python."